dev/Repeater/UniformGridLayoutState.cpp

```diff
@@ -36,15 +36,21 @@ void UniformGridLayoutState::EnsureElementSize(
     const winrt::UniformGridLayoutItemsStretch& stretch,
     const winrt::Orientation& orientation,
     double minRowSpacing,
-    double minColumnSpacing)
+    double minColumnSpacing,
+    unsigned int maxItemsPerLine)
 {
+    if (maxItemsPerLine == 0)
```

Comment:
Should we perhaps throw in this case instead of forcing it be 1 ?

Reply:
We could.  I was trying to mimic the behavior of WrapGrid (and I think) ItemsWrapGrid where it doesn't complain about 0 and still shows at least 1.